### ISSUE TITLE
Change behavior of FWD/RWD to move by program instead of block

### ISSUE THREAD

**Issue by factus10** [timex-sinclair-projects/TS-Pico#29]
### Discussed in https://github.com/timex-sinclair-projects/TS-Pico/discussions/18

<div type='discussions-op-text'>

<sup>Originally posted by **factus10** August  7, 2024</sup>
@ryangray idea to change from moving by program unit instead of by internal block.</div>

**Comment by ryangray** [comment]
I had sent modified versions to Ricardo that preserved the current behavior for `CODE n` to move by n blocks and adds `CODE n,1` for moving by n files. It also had `CODE 0,m` to move absolutely to file m, but that might be better as a separate SEEK command.

**Comment by factus10** [comment]
I seem to recall that he's added them to the current version.

**Comment by rcalcagno** [comment]
No, unfortunately I didn't include those in the last release; together with the other enhancement such as get/set log level and verbose, and DIR sorting. There are minor quircks to be addressed, such as definitive syntax(behavior of REW/FFW, and issuing DIR CODE parameter (DIR is a LOAD commands, so can't take CODE parameters) that prevented the added commands. Once this small issues are defined, the new code can be incorporated and tested.
